tests/integration/scenarios/dash_live.js

```diff
@@ -108,6 +108,7 @@ describe("dash live SegmentTimeline content", function () {
     expect(typeof videoRepresentation.index).to.equal("object");
 
     const videoRepresentationIndex = videoRepresentation.index;
+    const initVideoSegment = videoRepresentationIndex.getInitSegment();
 
     const initVideoSegment = videoRepresentationIndex.getInitSegment();
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Correct this attempt to modify "initVideoSegment" or use "let" in its declaration. ([javascript:S3500](https://sonarcloud.io/organizations/rx-player/rules#rule_key=javascript%3AS3500))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=249&issues=AWQT3116YwZU9vvrYAoj&open=AWQT3116YwZU9vvrYAoj)

[ISSUE_KEY]: AWQT3116YwZU9vvrYAoj